dev/tasks/verify-rc/github.Almalinux.amd64.yml

```diff
@@ -0,0 +1,25 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+
+jobs:
+  verify:
+    name: "Verify release candidate AlmaLinux"
+    runs:
+      using: 'docker'
+      image: 'AlmaLinuxDockerfile'
+
```

Comment:
If we don't work on Crossbow task in this pull request, we should not add/change any files under `dev/tasks/`.

Reply:
ok

Reply:
You want to work on Crossbow task in this pull request, right?

We should add `almalinux-verify-rc-*` and `ubuntu-verify-rc-*` entries to `docker-compose.yml` and use it in `dev/tasks/`. I think that it will be something like the following:

```yaml
almalinux-verify-rc-source:
  image: almalinux:${ALMALINUX}
  volumes:
    - .:/arrow:delegated
  command: >
    /bin/bash -c "
      /arrow/ci/dev/release/setup-rhel-rebuilds.sh &&
      /arrow/ci/dev/release/verify-release-candidate.sh source ${VERSION} ${RC}"
```

Note that we also need to add `ALMALINUX=8` to `.env`.

We can use this by `archery docker run -e VERSION=6.0.1 -e RC=1 almalinux-verify-rc-source`.

Reply:
I don't know why you marked this as resolved.
If you don't work on Crossbow task in this pull request, please revert changes under `dev/tasks/`.

Reply:
Sorry missed this.